webapp/app/appErrors/appErrors.js

```diff
@@ -31,15 +30,15 @@ const AppError = ({ error, closeAppError }) => {
 
 const AppErrors = ({ errors, closeAppError }) => (
   <TransitionGroup className={`app-errors${R.isEmpty(errors) ? ' hidden-transition' : ''}`} enter={true} appear={true}>
-    {errors.map(error => (
+    {errors.map((error) => (
       <CSSTransition key={error.id} timeout={500} classNames="fade">
         <AppError error={error} closeAppError={closeAppError} />
       </CSSTransition>
     ))}
   </TransitionGroup>
 )
 
-const mapStateToProps = state => ({
+const mapStateToProps = (state) => ({
   errors: ErrorsState.getAppErrors(state),
 })
```

Comment:
and replacing this by `useSystemServiceErrors()`

Reply:
better ```useServiceErrors```